dev_tools/scripts/make_keys.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
I'm sure we have a handful of these scripts lying around, and instead of fixing the details in each and every one every time it comes up for review, I'd like to have one single script in a merged repo that we fix up. This script does not, for example, support the use of `"` or any substring special to `sed` in passwords.  It also lacks quoting of a few expansions and a tiny bit of unnecessary code duplication. But since this is stuff that only I care about, we can fix that in a refactoring sprint.